backend/internal/models/service_test.go

```
@@ -108,38 +168,67 @@ func TestLegacyPublicIDCompatibilityCreatesAddressableInstance(t *testing.T) {
+		t.Fatal(err)
+	}
+	if abs, err := s.ModelAbsolutePath(m); err != nil || abs != path {
+		t.Fatalf("relative resolution=%q err=%v", abs, err)
+	}
+	if quantFromName("foo-q8_0.gguf") != "Q8_0" || quantFromName("foo.BF16.gguf") != "BF16" || quantFromName("none.gguf") != "" {
+		t.Fatal("quantization parsing mismatch")
+	}
+	if boolInt(true) != 1 || boolInt(false) != 0 {
+		t.Fatal("boolInt mismatch")
+	}
+	if newID() == "" || newID() == newID() {
```

golangci-lint (2.13.2)

[error] SA4000: identical expressions on the left and right side of the '==' operator

(staticcheck)